tests/scripts/test_pkgdev_manifest.py

```diff
@@ -44,6 +44,28 @@ def test_atom_target(self, repo, capsys, tool):
         matches = [x.cpvstr for x in repo.itermatch(options.restriction)]
         assert matches == ['cat/pkg-0']
 
+    def test_if_modified_target(self, repo, make_git_repo, tool):
```

Comment:
I think you should also check:

- what happens when ebuild is removed (but other ebuilds remain)
- what happens if all ebuilds in a package are removed (i.e. if it doesn't crash somewhere)